lisp/mastodon-toot.el

```diff
@@ -142,14 +141,53 @@ Set `mastodon-toot--content-warning' to nil."
       (mastodon-http--triage response
                              (lambda () (message "Toot toot!"))))))
 
+(defun mastodon-toot--get-acct()
+  "Adds the acct for the handle under the current point to the Kill ring.
+
+This only works if the html received is formatted to contain the domain. 
+it wont work for copying accts from the by-line or if someone entered a 
+handle manually"
+  (interactive)
+  (let ((user-address (plist-get (text-properties-at (point)) 'shr-url)))
+    (if (stringp user-address)
+        (let* ((split-address (split-string-and-unquote user-address "/"))
+               (domain (elt split-address 1))
+               (username (string-remove-prefix "@" (car (last split-address))))
+               (acct (concat "@" username "@" domain)))
+          (kill-new acct)
+          (message (format "Added %s to the kill ring." acct))
+          t)
+      (progn (message (format "No properly formated acct at point:%s." (point)))
+             nil))))
+
+(defun mastodon-toot--process-local (acct)
+  "Adds domain to local ACCT and replaces the curent user name with \"\"."
```

Comment:
I don't understand this comment.

Does `acct` have to be a local name?  What is the "current user name"?